src/ResourceManager/Network/ChangeLog.md

```diff
@@ -20,6 +20,10 @@
 ## Current Release
 
 ## Version 4.4.1
+* Added cmdlet to list available internet service providers for a specified Azure region
```

Comment:
@irrogozh please move this snippet you added to be under the `## Current Release` header ☺️

Reply:
Done